src/wp-includes/formatting.php

```diff
@@ -2758,7 +2758,7 @@ function trailingslashit( $string ) {
  * @return string String without the trailing slashes.
  */
 function untrailingslashit( $string ) {
-	return rtrim( $string, '/\\' );
+	return rtrim( (string) $string, '/\\' );
```

Comment:
In this case, as there are no type checks against `$string`, it could be anything, including an `array`, which would give an `Array to string conversion` warning.

An alternative is just to return `''` when `$string` is not a `string`. See [here](https://3v4l.org/QUJc6#v8.1.2).

```suggestion
	return is_string( $string ) ? rtrim( $string, '/\\' ) : '';
```